src/components/card/z-card-footer/index.tsx

```diff
@@ -26,16 +26,50 @@ export class ZCardFooter {
 
   @State() isOpen: boolean = false;
 
+  @State() allowTooltipAuthors: boolean = false;
+  @State() allowTooltipIsbn: boolean = false;
+
+  private ellipsisAuthors?: HTMLElement;
+  private ellipsisIsbn?: HTMLElement;
+
   @Listen("toggleClick")
   handleToggle(): void {
     this.isOpen = !this.isOpen;
+    setTimeout(() => {
```

Comment:
Instead of `setTimeout` you could add this line to the footer element (line 79): `onTransitionEnd={() => this.elementsEllipsis()}`
I also think this `setTimeout` is not really necessary because the animation is really fast, but it's ok of course.

Reply:
I really like your suggestion, the onTransitionEnd event it's the piece of code I was looking for throwing away the setTimeout function. By the way, concerning the current solution, I'm sorry to tell you that timeout it's mandatory otherwise it would not works, I tested it with both chrome and firefox browser

Reply:
Ah, you're right, because the code was executed too soon. :+1: